### PR TITLE
Add a dbWriteTable() method implementation for PrestoConnection

### PR DESCRIPTION
**Pull request by jarodmeng**

### CODE REVIEW

**Comment by jarodmeng** [comment]
Contributing towards #154

**Comment by jarodmeng** [comment]
All unit tests passed and R CMD CHECK returns no error/note/warning

File: R/dbWriteTable.R [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  conn, name, value,
+  overwrite = FALSE, ...,
+  append = FALSE, field.types = NULL, temporary = FALSE, row.names = NA
+) {
+  stopifnot(
+    length(overwrite) == 1 &&
+      is.logical(overwrite) &&
+      !is.na(overwrite)
+  )
+  stopifnot(is.data.frame(value))
```
Comment (leepface) [inline R/dbWriteTable.R:32]: inherits(value, "data.frame")

File: R/dbWriteTable.R [hunk 2]
```diff
@@ -0,0 +1,89 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+#' @include PrestoConnection.R
+NULL
+
+random_name <- function(n = 10) {
```
Comment (leepface) [inline R/dbWriteTable.R:10]: Meant to delete this?

**Review by leepface — CHANGES_REQUESTED** [review]

File: R/dbWriteTable.R (hunk 2 above)
Comment (jarodmeng) [reply R/dbWriteTable.R:10]: Good catch! Removed.

File: R/dbWriteTable.R (hunk 1 above)
Comment (jarodmeng) [reply R/dbWriteTable.R:32]: The function body of [base::is.data.frame()](https://github.com/wch/r-source/blob/trunk/src/library/base/R/dataframe.R#L138) is literally `inherits(x, "data.frame")`. Why is it necessary to replace the function call with its body?

Comment (leepface) [reply R/dbWriteTable.R:28]: Of course it is...R is so weird.

**Review by leepface — APPROVED** [review]

**Pull request merged**